crates/biome_module_graph/src/js_module_info/module_resolver.rs

```diff
@@ -54,7 +55,7 @@ pub struct ModuleResolver {
     /// "Module 0" derives its name from being stored at the first index of this
     /// vector. Other modules are those imported by module 0, either directly
     /// or transitively.
-    modules: Vec<JsModuleInfo>,
+    pub modules: Vec<JsModuleInfo>,
```

Comment:
Is it "right" to expose modules as is? Should we expose some APIs instead?

Reply:
Reverted, because this was only necessary for the `module_0_types()` method.